components/popover_list_members/index.ts

```diff
@@ -33,9 +56,9 @@ function makeMapStateToProps() {
     const doGetProfilesInChannel = makeGetProfilesInChannel();
     const doSortUsersByStatusAndDisplayName = makeSortUsersByStatusAndDisplayName(doGetProfilesInChannel);
 
-    return function mapStateToProps(state, ownProps) {
+    return function mapStateToProps(state: GlobalState, ownProps: Props) {
         const stats = getAllChannelStats(state)[ownProps.channel.id] || {};
-        const users = doGetProfilesInChannel(state, ownProps.channel.id, true);
+        const users = doGetProfilesInChannel(state, ownProps.channel.id, {active: true});
```

Comment:
I assume true is referring to `active` in `Filters` type. Please let me know if my assumption is correct

Reply:
👍🏼

Reply:
Is this a change in behaviour from the previous version? It looks like passing `true` might have been the same as not filtering the result here, so changing it to `{active: true}` might start filtering out inactive users.

Reply:
Actually, this had no effect. As discussed in other comment `users` prop wasn't being used in the component so I removed it from here as well.